samples/dfu/nrf5340/main.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/device.h>
+#include <lte_uart_dfu.h>
+#include <zephyr/logging/log.h>
+#include <dfu/dfu_target_mcuboot.h>
+#include <sample_interface.h>
+LOG_MODULE_REGISTER(app, CONFIG_APP_LOG_LEVEL);
+
+K_SEM_DEFINE(exit_sem, 0, 1);
+
+static uint8_t buf[2100];
```

Comment:
What is special about 2100? How was that number calculated?

Reply:
CONFIG_DOWNLOAD_CLIENT_BUF_SIZE + some change

Reply:
An idea could maybe be to use `CONFIG_DOWNLOAD_CLIENT_BUF_SIZE  + MY_DEFINE` where `MY_DEFINE` could be a reasonable name for a define which captures `some changes`. Maybe something like `#define BUFFER_OVERHEAD`